adb_graphics/datahandler/gribdata.py

```diff
@@ -258,217 +157,122 @@ def get_transform(self, transforms, val):
-                name = name.get(self.filetype)
-
-        if name is None:
-            print(f"Cannot find ncl_name for: ")
-            for key, value in spec.items():
-                print(f'{key}: {value}')
-            raise KeyError
-
-        # The level_type for the entire atmosphere could be L10 or L200. Thanks
-        # Grib2! Handle that in "try" statement when reading file.
-
-        name = name if isinstance(name, list) else [name]
-
-        try_name = ''
-        for try_name in name:
-            try_name = try_name.format(fhr=self.fhr,
-                                       grid=self.grid_suffix,
-                                       level_type=self.level_type)
-
-            try:
-                self._get_field(try_name)
-            except errors.GribReadError:
-                continue
-            else:
-                return try_name
-
-        msg = f'Could not find any of {try_name} in input file'
-        raise errors.GribReadError(msg)
-
-    def numeric_level(self, index_match=True, level=None, split=None):
-
-        '''
-        Split the numeric level and unit associated with the level key.
-
-        A blank string is returned for lev_val for levels that do not contain a
-        numeric, e.g., 'sfc' or 'ua'.
-        '''
-
-        level = level if level else self.level
-
-        # Gather all the numbers in the string
-        lev_val = ''.join([c for c in level if (c in digits or c == '.')])
-
-        # Convert the numbers to a list, and make integers or floats
-        if lev_val:
-            if split is not None:
-                lev_val = [int(lev) for lev in lev_val]
-            else:
-                lev_val = [float(lev_val) if '.' in lev_val else int(lev_val)]
-
-        # Gather all the letters
-        lev_unit = ''.join([c for c in level if c in ascii_letters])
-
-        if index_match:
-            if lev_unit == 'cm':
-                lev_val = [val / 100. for val in lev_val]
-            if lev_unit in ['mb', 'mxmb']:
-                lev_val = [val * 100. for val in lev_val]
-            if lev_unit in ['in', 'km', 'mn', 'mx', 'sr']:
-                lev_val = [val * 1000. for val in lev_val]
-
-        return lev_val, lev_unit
+    def latlons(self) -> list[np.ndarray]:
+        """Returns the set of latitudes and longitudes."""
+
+        coords = sorted(
+            [
+                str(c)
+                for c in list(self.field.coords)
+                if any(ele in str(c) for ele in ["lat", "lon"])
+            ]
+        )
+        lat = self.field.coords[coords[0]].to_numpy()
+        if len(lat.shape) == 1 and lat[-1] < lat[0]:
+            lat = lat[::-1]
+        lon = self.field.coords[coords[-1]].to_numpy()
+        return [lat, lon]
 
     @staticmethod
-    def opposite(values, **kwargs):
-    # pylint: disable=unused-argument
-
-        ''' Returns the opposite of input values  '''
+    def opposite(values: DataArray, **kwargs) -> DataArray:  # noqa: ARG004
```

Comment:
```suggestion
    def opposite(values: DataArray, **_kwargs) -> DataArray:
```
I'm not sure, but I think preceding the argument name with an underscore will be  enough for `ruff` to understand that you are intentionally not using that argument.

Reply:
Oooh! I'll try it out.